website/docs/cloud-docs/api-docs/state-versions.mdx

```diff
@@ -72,14 +72,15 @@ This POST endpoint requires a JSON object with the following properties as a req
-| `data.attributes.state`          | string  |           | Base64 encoded raw state file                                                                                                                                                                             |
-| `data.relationships.run.data.id` | string  | (nothing) | **Optional** The ID of the run to associate with the state version.                                                                                                                                       |
+| Key path                             | Type    | Default   | Description                                                                                                                                                                                                                 |
+|--------------------------------------|---------|-----------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `data.type`                          | string  |           | Must be `"state-versions"`.                                                                                                                                                                                                 |
+| `data.attributes.serial`             | integer |           | The serial of the state version. Must match the serial value extracted from the raw state file.                                                                                                                             |
+| `data.attributes.md5`                | string  |           | An MD5 hash of the raw state version                                                                                                                                                                                        |
+| `data.attributes.lineage`            | string  | (nothing) | **Optional** Lineage of the state version. Should match the lineage extracted from the raw state file. Early versions of terraform did not have the concept of lineage, so this is an optional attribute.                   |
+| `data.attributes.state`              | string  |           | Base64 encoded raw state file                                                                                                                                                                                               |
+| `data.attributes.json-state-outputs` | string  | (nothing) | **Optional** Base64 encoded output values as represented by `terraform show -json` (the contents of the values/outputs key). If omitted, Terraform Cloud will populate the workspace outputs from state after a short time. |
```

Comment:
```suggestion
| `data.attributes.json-state-outputs` | string  | (nothing) | **Optional** Base64 encoded output values as represented by `terraform show -json` (the contents of the values/outputs key). If omitted, Terraform Cloud populates the workspace outputs from state after a short time. |
```

Suggest changing "Terraform Cloud will populate..." to "Terraform Cloud populates..." to avoid future tense.